### PR TITLE
Question 2 and 3 modified

### PR DESCRIPTION
**Pull request by AmanMathur23**

### CODE REVIEW

**Review by mudgalabhay — COMMENTED** [review]
Thanks for Q2, 3
1. In Homework.ipynb file there is one error it is showing when experimenting with kernel_size=1, num_kernels=[16, 32, 64, 128, 256], learning_rate=0.01, Do we need to include this in final file?
2. For Q1 I had done without using pytorch initially(normalization part) can we include that one, will that work with Q2 & 3?, let me know if you need any modification in Q1 according to Q2&3
